src/docs/product/sentry-basics/guides/getting-started/index.mdx

```diff
@@ -1,128 +1,133 @@
 
-Sentry has built-in integrations with:
+Sentry has built-in SCM integrations with:
 
+- [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/)
 - [GitHub](/product/integrations/source-code-mgmt/github/)
 - [GitLab](/product/integrations/source-code-mgmt/gitlab/)
 - [Bitbucket](/product/integrations/source-code-mgmt/bitbucket/)
 
-If you're using a different SCM provider or don't want Sentry to connect to your
-repository, check out our
-[releases documentation](/product/releases/setup/manual-setup-releases/)
-to learn how you can still enable most of these features (#1 - #3) by telling
-Sentry about your commit metadata.
+If you're using a different SCM provider or don't want Sentry to connect to your repository, check out our [Releases documentation](/product/releases/setup/manual-setup-releases/) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.
 
-### 3.3. Issue Tracking
+### Issue Tracking {#33-issue-tracking}
 
-Enabling the 2-way integration with your Issue Tracking solution allows you to create a new issue from within the Sentry issue page, or link it to an existing one. In addition to **GitHub**, **GitLab**, and **Bitbucket** Issues that are enabled with the respective SCM integration, Sentry also integrates with: [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/), [Clubhouse](/product/integrations/project-mgmt/clubhouse/), [JIRA](/product/integrations/project-mgmt/jira/), and others.
+Enabling an integration with your issue tracking solution allows you to create a new issue from within the **Issue Details** page in [sentry.io](https://sentry.io), or link to an existing one. In addition to GitHub, GitLab, and Bitbucket issues that are associated with the respective SCM integration, Sentry also integrates with [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/), [Clubhouse](/product/integrations/project-mgmt/clubhouse/), [Jira](/product/integrations/project-mgmt/jira/), and others.
 
-> ![JIRA 2-Way Integration](jira-2way-binding.gif)
->
-> 2-Way integration with JIRA issue tracking.
+For a list of all supported integrations, check out out our [full Integrations documentation](/product/integrations/).
 
-For a full list of all supported integrations, go to our documentation on [Integrations](/product/integrations/).
+While you can manually link Sentry issues to issues in your issue tracking solution, for [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/#automatically) and [Jira](/product/integrations/project-mgmt/jira/#automatically), you can set up automated issue management when you create alerts, so that external issues are created for new Sentry issues on your behalf. When you set up this automation, Sentry retrieves the fields configured in your Azure DevOps or Jira workspace so that you can set how these fields should be populated.
```

Comment:
Same thing here:
> While you can manually link Sentry issues to issues in your issue tracking solution, for Azure DevOps and Jira, you can set up automated issue management when you create alerts, so that external issues are created for new Sentry issues on your behalf.


Maybe something like 
```
You can set up automated issue management when you create alerts for Azure DevOps and Jira. With will create external issues for new Sentry issues on your behalf. For other issue tracking solutions, you can manually link Sentry issues.
```